homeassistant/components/zha/core/device.py

```diff
@@ -143,7 +150,9 @@ def device_info(self):
             IEEE: ieee,
             ATTR_MANUFACTURER: self.manufacturer,
             MODEL: self.model,
-            NAME: self.name or ieee
+            NAME: self.name or ieee,
+            QUIRK_APPLIED: self.quirk_applied,
```

Comment:
I don't think arbitrary keys are read for device info. Are they?

Reply:
This is on zha device... this will get used by the API not HA